File: src/API/WesternStatesWater.WestDaat.Common/DataContracts/OrganizationMemberUpdateRequest.cs
namespace WesternStatesWater.WestDaat.Common.DataContracts;

public class OrganizationMemberUpdateRequest : OrganizationStoreRequestBase
{
    required public Guid OrganizationId { get; set; }

    required public Guid UserId { get; set; }

    required public string Role { get; set; }
}

File: src/DashboardUI/src/data-contracts/OrganizationMemberUpdateRequest.ts
import { Role } from "../config/role";
import { OrganizationStoreRequestBase } from "./OrganizationStoreRequestBase";

export interface OrganizationMemberUpdateRequest extends OrganizationStoreRequestBase {
  $type: 'OrganizationMemberUpdateRequest';
  organizationId: string;
  userId: string;
  role: string;
}
